api-scripts/patron/placing-holds/README.md

```diff
@@ -0,0 +1,38 @@
+## API test:
```

Comment:
Thanks for placing this set of script in a separate section called "api-scripts".  @javalkar-varun, @Svitlana-Zmiivska  Any thoughts on having a separate API scripts section? I think it's a good idea because this test explicitly targets an API call, but at the same time it looks like a simple workflow to me.

Reply:
It seems to me it might be a good idea to separate API tests from workflows. What if I create a separate folder test and move workflows-scripts and api-scripts folders there?

Reply:
API test section makes sense because this is not a workflow test.

Reply:
We will keep the API section then.